spec/models/subscription_spec.rb

```diff
@@ -204,4 +204,18 @@
       end
     end
   end
+
+  describe 'expired?' do
+    context 'when the expires_on date is in the future' do
+      let(:subscription) { create(:daily_subscription, expires_on: Time.zone.today + 1.month) }
```

Comment:
We could use `build_stubbed` in this spec instead of `create` since we don't need to actually create the record?